schema/deploy/shipping/views.sql

```diff
@@ -844,4 +844,73 @@ comment on view shipping.hcov19_observation_v1 is
   'Custom view of hCoV-19 samples with presence-absence results and best available encounter data';
 
 
+create or replace view shipping.scan_return_results_v1 as
+    select distinct on (barcode, lineage)
+        sample_id,
+        barcode as qrcode,
+        encountered as collect_ts,
+        case
+            when sample_id is null then 'not-received'
+            when sample_id is not null and present is null then 'pending'
```

Comment:
I think `sample.sample_id is not null and presence_absence_id is null` is more specific for pending. Or else this will never reach 'inconclusive' case

Reply:
Ah right, I'm using `presence_absence_id` to check for a test result anyway, so using that instead of a presence result is also more consistent.